storybook/stories/CustomerDetails/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
```

Comment:
Would like to comment on other components that are not here.

For `CustomerAppeal - ReasonSelectionCard`, can change `Lost/stolen token` to `Lost token` as well. And since it will be confirmed soon, I think it's great to add a `Defective token` reason here as well.

For `DarkButton` and `Checkbox`, can add a margin between components so that they are not touching one another.

Reply:
what is `Defective token` mean. is it new reason type ?

Reply:
updated `ReasonSelectionCard` and `Checkbox`

Reply:
`DarkButton` is updated in atom level [PR319](https://github.com/rationally-app/mobile-application/pull/319/files#diff-dd5ba395728d68b0c73183cb491783fd6b12afc9c580e40b041c50eaa29c2790R74-R79)

Reply:
FYI, `DarkButton` PR was merged